docs/edge/introduction.rst

```diff
@@ -0,0 +1,344 @@
+  sufficient compute to run the CrateDB instances desired. We recommend the use
+  of a K8s autoscaler. For reference, the CrateDB Edge stack - without Grafana
+  or Loki - requires about 2 CPUs and 2500 Mi of memory. We recommend at least
+  4 CPUs per node for reliable performance.
+
+* A Kubernetes version 1.20, 1.21, or 1.22;
+
+* A storage class for persisting your data (SSD recommended). As outlined in
+  the installation script, CrateDB Edge expects the storage classes
+  ``crate-premium`` (SSD) and ``crate-standard`` (SSD or Spindle). Also ensure
+  that you are able to set the field ``allowVolumeExpansion`` to ``true``.
+
+* Unless you are experienced with operating Kubernetes clusters, we recommend
+  to start with a dedicated Kubernetes cluster for CrateDB Edge. If you use an
+  existing Kubernetes setup, be aware that the following components will be
+  set up during installation of CrateDB Edge, which may interfere with your
+  existing configuration:
+
+  * cert-manager (version 1.6.1)
+  * ingress-nginx (version 1.2.3)
+  * Optionally, Grafana, Loki, or Promtail (via Helm).
+
+Beyond this, using the CrateDB Cloud stack requires creating a CrateDB Cloud
+account and an organization, which will become the owner of the Edge region in
+which the cluster can be deployed. One must also access the CrateDB Cloud
+Console in order to deploy the cluster itself, using the provided script. These
+steps will be explained below.
+
+Some Kubernetes knowledge, especially regarding networking and storage, is
+recommended when using CrateDB Edge, especially when using it as-is. If you're
+uncertain, you may benefit from using CrateDB Edge in combination with
+:ref:`cloud providers or third-party tools <edge-providers>` as described
+further down in this tutorial.
+
+.. NOTE::
+    A special note about bare metal Kubernetes clusters: CrateDB Edge should
+    work on any bare metal cluster, but the CrateDB instances running within
+    require a load balancer for outside access. If you do not have a load
+    balancer (for example `MetalLB`_), you can still access the CrateDB
+    clusters within, but you will need to figure out the node ports to use.
+
+
+.. _edge-signup:
+
+Sign up
+-------
+
+To use the CrateDB Cloud software, you must first sign up. Follow the steps
+outlined in :ref:`this tutorial <sign-up>` to do so.
+
+
+.. _edge-create-org:
+
+Create an organization
+----------------------
+
+When you first log in to the CrateDB Cloud Console after having created an
+appropriate account, you will arrive at the Organization overview page. Here
+you will be prompted to create an organization.
+
+.. image:: ../_assets/img/cloud-create-org.png
+   :alt: CrateDB Console organization creation screen
+
+Fill out the name of the organization and click the *Create organization*
+button. After a short moment, the organization will be created and you can
+proceed.
+
+You will be taken to the Subscriptions tab of the Organization overview page.
```

Comment:
The subscriptions tab has been deleted now and so this is no longer true. The user will be taken to the `dashboard`  tab after creating an organization